### DISCUSSION TITLE
Renaming Abilities

### DISCUSSION THREAD

**Discussion by HappyPigUK** [JohanFalt/Foundry_WoD20#702] [Q&A] [answered]
Sorry if this has already een raised in Issues or even in Discussions, i couldnt find any reference and i didnt want to raise it just in case.

When it comes to renaming an Abilities and rolling it references the old name, for example on a Vampire sheet i changed the Knowledge skill Gremmayre to Koldunism (Cant use a secondary skill with a Dicipline roll) but when i roll a Koldunism power it shows as Gremayre, while its not game breaking its a tad annoying.

So is there away to force it to use the new name or is that feature just not available yet?

Thanks.

**Comment by JohanFalt** [comment]
Is this in the chat this comes up?

**Comment by HappyPigUK** [comment]
Aye, in the chat window.

**Comment by JohanFalt** [comment, marked as answer]
Then it is a bug. I’ll add it to the list for version 3.2

**Reply by HappyPigUK** [reply]
Thank you very much, also i realy appreciate the work you have done on this, makes my games a lot easier to run. Thank you.
